dbschema/partnership-producing-medium.esdl

```diff
@@ -0,0 +1,7 @@
+module default {
```

Comment:
I'd probably put this in the `Product` module. Since that's where Medium is. This is just saying which partners are producing each medium.

Reply:
Would you still split it into this separate `.esdl`, or you want the whole thing to move into the `Product.esdl`?

Reply:
I'd keep it separate